tactics/tacticals.mli

```diff
@@ -209,6 +209,7 @@ module New : sig
   val tclMAP : ('a -> unit tactic) -> 'a list -> unit tactic
 
   val tclTRY : unit tactic -> unit tactic
+  val tclTRYb : unit tactic -> bool list tactic
```

Comment:
This probably deserves a line a comment at least to remind the client which of success or non-success is `true` and which is `false`.

It's pretty nice. It's in case like this that we miss Haskell-style type classes (we could make a `Monoid 'a => tclDISPATCHM : 'a tactic list -> 'a tactic`. Ah well :) .